src/main/java/org/apache/mesos/hdfs/config/ConfigServerException.java

```diff
@@ -0,0 +1,26 @@
+package org.apache.mesos.hdfs.config;
+
+/**
+ * Indicates a failure to startup the config service, likely a jetty failure
+ */
+public class ConfigServerException extends RuntimeException {
+
+  private String message;
+
+  public ConfigServerException(String msg) {
+    this.message = msg;
+  }
+
+  public ConfigServerException() {
+    this("");
+  }
+
+  @Override
```

Comment:
Why override the message field and getMessage? Doesn't RuntimeException (Throwable, really) already have a message field? Or is it just so you can add a setMessage method? Why?